docs/src/pages/guides/minimizing-bundle-size/minimizing-bundle-size.md

```diff
@@ -136,30 +103,40 @@ Pick one of the following plugins:
 
   module.exports = {plugins};
   ```
+
+If you are using Create React App, there is no need for a `.babelrc.js` file and the configuration is easy by using [craco](https://github.com/sharegate/craco):
```

Comment:
Please remove the `easy` here. It rarely helps stating in the documentation that something is easy. Especially if they're written by the creators of the tool in question.